docs/can-canjs/can-api.md

```diff
@@ -277,17 +294,39 @@ console.log(areSomeComplete); //-> false
 ```
 @codepen
 
-<details>
-<summary>Infrastructure APIs</summary>
+</details>
+
+## Typed properties
+
+[can-type] can be used to define typed properties in [ObservableObject] and [StacheElement]
+- Types are strict by default, which means an error will be thrown if a property is set to a value of the wrong type.
+- [can-type/convert] can be used to create a property that will always convert its value to a specific type.
+- [can-type/maybe] can be  used to create a property that can be `null` or `undefined` as well as whatever valid values the type allows
+that has a `person` property with `Person` type:
```

Comment:
Is this left by accident?

![image](https://user-images.githubusercontent.com/5851984/69753479-52d88e80-1119-11ea-8193-3bec7c1eb2fc.png)